dips/dip-158.md

```diff
@@ -0,0 +1,795 @@
+
+This payment type is useful for delayed fulfillment or pre-authorizing an expected amount to ensure that an amount can be captured after services are rendered. 
+
+Off-chain negotiations for Auth / Capture payment are divided into two steps accordingly. First, off-chain negotiations will result in funds being authorized, i.e. locked by the wallet. Later, a separate off-chain negotiation triggered by the receiving VASP will capture the funds. Note that for the time being, multi-capture is not supported and there can be only a single capture request.
+
+Successful authorization will not necessarily be followed by a successful capture. It is possible that the receiver will choose to cancel the authorization and essentially void the payment before it was captured. One possible reason to void a payment is that the products for which an amount was locked are out of stock and cannot be fulfilled. 
+
+---
+## New Elements and Existing Elements Extensions
+---
+This section specifies extensions for existing elements (e.g. new values) and introduces new ones required for P2M scenarios.
+
+
+## PaymentActorObject Extension
+The PaymentActorObject is extended to include the field `merchant_data` of type `MerchantDataObject`.
+Field|Type|Required?|Description|
+|-|-|-|-|
+|merchant_data|MerchantDataObject|N|This element is required for some scenarios. It includes the details of the merchant to be displayed to the payer|
+|
+
+## PaymentActor Status Enum Extension
+The following values are added to the status enum of [PaymentActor](https://dip.diem.com/dip-1/#paymentactorobject) object:
+
+* `authorized` - This means the actor has performed all required tasks and funds are locked for a future capture. Note that the actual lock is done by the wallet (sender), so for the receiver this status means that all risk checks were successful and it is willing to accept the payment. 
+* `needs_init_data` - This means the actor requires basic data to initialize the payment. For the receiver, this status means that the payer data is required. For the sender, it means that the merchant data is required  
+
+For completeness purposes, the full list of values is: 
+* `none`
+* `needs_kyc_data`
+* `ready_for_settlement`
+* `abort`
+* `authorized`
+* `soft_match`
+* `needs_init_data`
+
+Allowed transition relevant for P2M:
+
+| Current Status | Next Status | Transition Comments |
+|-|-|-|
+| none | needs_init_data |Initial transition. Most common for P2M scenarios in which the actor requires additional data to initialize the payment|
+| needs_init_data | ready_for_settlement | All required tasks were performed and the payment is ready for settlement |
+| needs_init_data | authorized | All required tasks were performed and the funds are ready to be locked |
+| authorized | ready_for_settlement | Funds captured |
+| authorized | abort | Payment was voided |
```

Comment:
recommend

| * | abort | Payment was voided |

I don't think we need to be explicitly labeling all the times when an abort can happen. We could move to abort from any state because the local or remote side has decided to abort the exchange.

Reply:
I agree we might use an asterisk for that, but we need a way to eliminate the option of voiding an auth by the customer (which doesn't make sense...)

Reply:
I agree that we could do that. 
However, I used the transitions table to document and clarify the different abort scenarios. I think it has value.
If there are no special objections, I will leave as is.